docker-compose.yml

```diff
@@ -70,6 +70,13 @@ services:
         image: "${DOCKER_USERNAME}/conangcc63:${DOCKER_BUILD_TAG}"
         container_name: conangcc63
         tty: true
+    conangcc64:
```

Comment:
So conangcc62 is not deprecated?

Reply:
I just removed from travis file, but you are right, it doesn't make any sense keep on docker-compose. I'll remove it.